### PR TITLE
feat: create balance component

### PR DESCRIPTION
**Pull request by vromeiro**
## Description

Create new balance component

Fixes # (issue)

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration.

## Screenshots (if appropriate):

https://github.com/ocean-ds/ocean-ios/assets/918251/7a3c4e34-6d51-42c9-bf6a-65e600291315

## Checklist:

- [x] I have read the [CONTRIBUTING](CONTRIBUTING.md) document
- [x] I have performed a self-review of my own code
- [x] My changes generate no new warnings
- [x] I have made corresponding changes to the documentation (if appropriate)
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] All new and existing tests passed

### CODE REVIEW

File: OceanComponents/Classes/Components/Balance/Ocean+BalanceCell.swift [hunk 1]
```diff
@@ -0,0 +1,372 @@
...
+                stack.axis = .horizontal
+                stack.alignment = .fill
+                stack.distribution = .fill
+                stack.spacing = Ocean.size.spacingStackXxs
+                stack.isSkeletonable = true
+
+                stack.add([
+                    eyeContainerView,
+                    titleStack,
+                    UIView(),
```
Comment (AcassioM) [inline OceanComponents/Classes/Components/Balance/Ocean+BalanceCell.swift:114]: duvida real (rsss): o que essa view faz?

Comment (vromeiro) [reply OceanComponents/Classes/Components/Balance/Ocean+BalanceCell.swift:114]: ela estica, quanto precisar, pra nao esticar os outros componentes

File: OceanComponents/Classes/Components/Balance/Ocean+BalanceCell.swift [hunk 2]
```diff
@@ -0,0 +1,372 @@
...
+            case .collapsed:
+                UIView.animate(withDuration: 0.3) {
+                    self.arrowView.transform = CGAffineTransform(rotationAngle: 0)
+                    self.headerDetailsStack.alpha = 0
+                } completion: { _ in
+                    self.headerDetailsStack.isHidden = true
+                }
+            case .expanded:
+                UIView.animate(withDuration: 0.3) {
+                    self.arrowView.transform = CGAffineTransform(rotationAngle: (180.0 * .pi) / 180.0)
```
Comment (AcassioM) [inline OceanComponents/Classes/Components/Balance/Ocean+BalanceCell.swift:351]: olha aê... o cara usando pi pra fazer conta... hahahahahah tooop

Comment (vromeiro) [reply OceanComponents/Classes/Components/Balance/Ocean+BalanceCell.swift:351]: 😎

**Review by renanmassaroto — APPROVED** [review]

File: OceanComponents/Classes/Components/Balance/Ocean+BalanceCell.swift (hunk 1 above)
Comment (AcassioM) [reply OceanComponents/Classes/Components/Balance/Ocean+BalanceCell.swift:114]: Renan me mostrou essa tecnica ai ontem... top demais

**Review by AcassioM — APPROVED** [review]

**Pull request merged**